engine/src/main/java/com/odysseusinc/arachne/executionengine/execution/sql/SQLService.java

```diff
@@ -251,7 +251,7 @@ public static class SqlOverseer extends AbstractOverseer {
         private final StringBuffer stdout;
 
         public SqlOverseer(long id, Instant started, StringBuffer stdout, CompletableFuture<ExecutionOutcome> result) {
-            super(id, (stage, out) -> {}, started, null, 0, result);
+            super(id, (stage, out) -> {}, started, null, 0, new StringBuffer(), result);
```

Comment:
Please extract meaningful constants for 'null' and 0

Reply:
When you see and read this line it says nothing, sorry

"super(id, (stage, out) -> {}, started, null, 0, new StringBuffer(), result);"

Reply:
This is pretty much the same with any method call. If you don't know what each parameter means, you need to go and find out. 
Do you want to require extracting a constant for null each time a null is used as method argument?
And how do you propose to name this constant?